android/src/main/java/com/reactlibrary/rnwifi/RNWifiPackage.java

```diff
@@ -26,6 +26,6 @@
 
     @Override
     public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
-        return Arrays.<ViewManager>asList();
+        return Collections.emptyList();
     }
```

Comment:
I cannot see why this is related to isRemoveWifiNetwork()?

Reply:
It's not at all. Just ran lint as there was a Lint error explained here https://github.com/JuanSeBestia/react-native-wifi-reborn/pull/80.